Simplified/Catalog/NYPLCatalogUngroupedFeed.m

```diff
@@ -84,6 +91,7 @@ - (instancetype)initWithOPDSFeed:(NYPLOPDSFeed *const)feed
     }
     [self.books addObject:book];
   }
+  self.booksFromLastFetchNotSupported = (self.books.count == 0) && (feed.entries.count > 0);
```

Comment:
if I understand well this means that we had some books in the feed but none of them is supported by SimplyE, correct? If so, do you think `noSupportedBooksInLastFetch` is more clear? I was struggling to understand the meaning of this property until I saw the code. 

I was also wondering if it can be a computed property instead -- I'm not sure that's ok though.

Reply:
Right, I need to have the naming convention tips open all the time.

I don't think it can be a computed property as we use the entries of `NYPLOPDSFeed` to determine the value, and we are not storing that in `NYPLCatalogUngroupedFeed`

Reply:
oh I see, you're right... sounds good.